easybuild/easyconfigs/e/ELPA/ELPA-2022.05.001-foss-2022a-CUDA-%(cudaver)s.eb

```diff
@@ -0,0 +1,44 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Authors::   Inge Gutheil <i.gutheil@fz-juelich.de>, Alan O'Cais <a.ocais@fz-juelich.de>
+# License::   MIT/GPL
+#
+##
+
+name = 'ELPA'
+version = '2022.05.001'
+versionsuffix = '-CUDA-%(cudaver)s'
```

Comment:
should be removed, as this is a non-CUDA easyconfig?

Reply:
you may also have to remove this easyconfig, because it has the `%(cudaver)s` template in its name

Reply:
ugh, I guess I forgot to remove the `versionsuffix` from the EasyConfig, and then `--new-pr` made this name for it. Strange... I'll clean it up :D